src/common-ui/components/IndexDropdown.js

```diff
@@ -74,10 +78,57 @@ class IndexDropdown extends PureComponent {
         return this.placeholder.toLowerCase()
     }
 
+    renderTags() {
+        const tagOptions = this.props.tags.map((tag, i) => (
+            <span className={styles.tagPill}>
+                {tag}
+                {
+                    <button
+                        onClick={this.props.handleTagSelection(i)}
+                        className={styles.cross}
+                    />
+                }
+            </span>
+        ))
+        return tagOptions
+    }
+
+    renderDomains() {
+        const domainOptions = this.props.tags.map((tag, i) => (
+            <span className={styles.domainPill}>
+                {tag}
+                {
+                    <button
+                        onClick={() => this.props.handleTagSelection(tag)}
+                        className={styles.cross1}
+                    />
+                }
+            </span>
+        ))
+        return domainOptions
+    }
+
     render() {
+        // console.log(this.placeholder)
         return (
             <div className={this.mainClass} ref={this.props.setTagDivRef}>
+                {!this.props.isForSidebar ? (
+                    <div className={this.styles.numberTags}>
+                        <span className={this.styles.bold}>
+                            {this.props.numberOfTags}
+                        </span>{' '}
+                        {this.unit} selected
+                    </div>
+                ) : null}
+                <span>
+                    {this.placeholder === 'Tags' ? this.renderTags() : null}
+                    {this.placeholder === 'Domains'
+                        ? this.renderDomains()
+                        : null}
+                </span>
+
                 <div
+                    style={{ background: '#fff' }}
```

Comment:
`This style rule might end with confusion being inlined; maybe better to move it to the CSS module.`